imports/plugins/core/accounts/client/templates/profile/userOrdersList.js

```diff
@@ -0,0 +1,82 @@
+import { Template } from "meteor/templating";
+import { Orders, Media } from "/lib/collections";
+import CompletedOrder from "/imports/plugins/core/checkout/client/components/completedOrder";
+
+/**
+ * @method handleDisplayMedia
```

Comment:
Isn't there a way we could be rendering the Completed Order container rather than duplicating all this code?